Week2task10/src/SubmitButton.java

```diff
@@ -0,0 +1,113 @@
+import java.awt.BorderLayout;
+import java.awt.Dimension;
+import java.awt.GridLayout;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.awt.event.KeyEvent;
+import java.awt.event.KeyListener;
+import java.awt.event.KeyAdapter;
+
+import javax.swing.Box;
+import javax.swing.BoxLayout;
+import javax.swing.JButton;
+import javax.swing.JComboBox;
+import javax.swing.JFrame;
+import javax.swing.JLabel;
+import javax.swing.JOptionPane;
+import javax.swing.JPanel;
+import javax.swing.JTextField;
+import javax.swing.WindowConstants;
+
+public class SubmitButton extends JFrame {
+	JFrame frame = new JFrame();
+	JButton submit = new JButton("Submit");
+	
+	public SubmitButton() {
+		super( "Learn IT Girl" );
+		frame.setDefaultCloseOperation(WindowConstants.EXIT_ON_CLOSE);
+		frame.setSize(400, 250);
+		frame.setLocation(200, 200);
+		frame.setLayout(new BorderLayout());
+		frame.setVisible(true);
+		
+		// the text field
+		JPanel panel = new JPanel();
+		JTextField nameField = new JTextField(30);
+		JTextField surnameField = new JTextField(30);
+		JTextField emailField = new JTextField(30);
+		JLabel name = new JLabel("Name: ");
+	    JLabel surname = new JLabel("Surname: ");
```

Comment:
Please add a space or tab here that this line be aligned with the line above.

Reply:
I have all of that lines aligned on my computer. I don't know, why online some lines aren't aligned.